pages/about/security.md

```diff
@@ -1,12 +1,14 @@
 ---
 permalink: /about/security/
-redirect_from:
-- /security/
-- /about/security-information/
 layout: styleguide
 title: Security
 category: About
 lead: Developing and using USWDS with security in mind
+subnav:
+  type: security_updates
```

Comment:
This ties the content in `security_updates` collection to Security in sidenav. Otherwise the current status doesn't display properly.